tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -6,6 +6,7 @@
 from tests.common.utilities import wait, wait_until
 from netaddr import IPAddress
 from tests.common.helpers.assertions import pytest_assert
+from tests.snmp.test_snmp_phy_entity import redis_get_keys
```

Comment:
don't import from another test, copy redis_get_keys method to some common folder